packages/react-openapi/src/OpenAPISpec.tsx

```diff
@@ -48,6 +48,7 @@ export function OpenAPISpec(props: { rawData: any; context: OpenAPIClientContext
                                 // Description of the parameter is defined at the parameter level
                                 // we use display it if the schema doesn't override it
                                 description: parameter.description,
+                                example: parameter.example,
```

Comment:
Why do we need this? Isn't `example` on the `parameter.schema` itself.

Reply:
To maintain compatibility with Swagger 2.0 (in our docs we say we support 2.0). In 2.0, `example` is not on `parameter.schema` but on `parameter` itself.

In 3.0, `example` is on `parameter.schema`, so the destructuring happening in the line below will take of it

Reply:
https://swagger.io/docs/specification/2-0/adding-examples/